community/README.md

```diff
@@ -0,0 +1,62 @@
+# Nebula Graph Community Architecture
+
+The Nebula Graph community is organized as shown below:
+
+![Nebula Community Architecture](./images/structure.png)
+
+## PMC
+
+PMC (Project Management Committee) is the entity the controls and leads the whole nebula project. PMC members have the right to propose a new Maintainers or Committers, and the right to vote for the major decisions for Nebula Graph. See [PMC List](./pmc-list.md) for a list of PMC members.
```

Comment:
is the entity that controls and leads the whole nebula project. 

PMC members have the right to propose a new Maintainer or Committer